hourglass.py

```diff
@@ -53,10 +54,13 @@ def load_and_filter_annotations(self, path_to_train_anns,path_to_val_anns, subse
         print(f"Train/Val dfs contains {len(train_df)}/{len(val_df)} anns")
         return train_df.reset_index(), val_df.reset_index()
 
-    def _start_train(self, batch_size, model_base_dir, epochs, initial_epoch, model_subdir, current_time, subset):
+    def _start_train(self, batch_size, model_base_dir, epochs, initial_epoch, model_subdir, current_time, subset, pickle_name):
         self._compile_model()
-
-        train_df, val_df = self.load_and_filter_annotations(DEFAULT_TRAIN_ANNOT_PATH, DEFAULT_VAL_ANNOT_PATH, subset)
+        # insert logic here for pickled dataframes
+        if pickle_name != None:
```

Comment:
Use `is not` instead of `!=`
"Comparisons to singletons like None should always be done with is or is not, never the equality operators."
[PEP 8 Style Guide](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)